xfields/fieldmaps/interpolated_src/atomicadd.h

```diff
@@ -0,0 +1,15 @@
+// copyright ################################# //
```

Comment:
Same as above

Reply:
`xt.general._pkg_root.joinpath('headers/atomicadd.h')` was already used in beambeam3d.py. I just removed these two lines from beambeam3d.h:
```
/include_file ../../fieldmaps/interpolated_src/atomicadd.clh for_context opencl
/include_file ../../fieldmaps/interpolated_src/atomicadd.h for_context cpu_serial cpu_openmp
```